.circleci/config.yml

```diff
@@ -4,69 +4,72 @@ aliases:
   - &working_directory ~/merchant-center-application-kit
 
   - &restore_yarn_cache
-    name: 'Restoring yarn cache'
-    keys:
-      - v1-yarn-cache-{{ checksum "yarn.lock" }}
-      - v1-yarn-cache
+    restore_cache:
+      name: 'Restoring yarn cache'
+      keys:
+        - v1-yarn-cache-{{ .Branch }}-{{ checksum "yarn.lock" }}
```

Comment:
React (and other CI configs that I found) use the `.Branch` part in the cache key name. I think it makes sense, so that branches don't mess with each other dependencies.